components/Dashboard/ReviewForm.tsx

```diff
@@ -75,12 +75,12 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
         {/* Design  */}
         {currentStepId === 'WELCOME_PAGE' && (
           <div className="relative w-full max-w-lg rounded-xl bg-white p-6 pt-4 shadow-lg">
-            <div className="absolute -top-3.5 right-3 flex items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
-              <HeartFilledIcon className="h-4 w-4" />
-              Powered by Review
-            </div>
+              <div className="absolute -top-3.5 right-3 flex dark:text-black items-center gap-2 rounded-full border bg-white px-4 py-1 text-sm">
```

Comment:
@hemantwasthere Can you please verify this?

Reply:
okay